packages/java-edition/src/mcfunction/mcdocAttributes.ts

```diff
@@ -32,11 +48,19 @@ const scoreHolderValidator = validator.alternatives<ScoreHolderConfig>(
 )
 
 export function registerMcdocAttributes(meta: core.MetaRegistry, rootTreeNode: mcf.RootTreeNode) {
-	mcdoc.runtime.registerAttribute(meta, 'command', () => undefined, {
-		// TODO: validate slash
+	mcdoc.runtime.registerAttribute(meta, 'command', commandValidator, {
 		// TODO: fix completer inside commands
-		stringParser: () => {
-			return mcf.command(rootTreeNode, parser.argument)
+		stringParser: ({ slash, empty }) => {
+			return (src, ctx) => {
+				if ((empty && !src.canRead()) || (slash === 'chat' && src.peek() === '/')) {
```

Comment:
Is this supposed to be `src.peek() !== '/'`?

Reply:
It looks like it yeah, good catch!